torch/onnx/symbolic_opset11.py

```diff
@@ -487,7 +487,17 @@ def _get_arange_dtype(dtype):
         dtype = sym_help._maybe_get_const(dtype, 'i')
         return dtype
 
-    if len(args) == 5:
+    if len(args) == 2:
+        # aten::arange(Scalar end, Tensor out)
+        type, end, start, step = sym_help._arange_cast_helper(g, end=args[0])
+        start_default = g.op("Constant", value_t=torch.tensor(0, dtype=sym_help.scalar_type_to_pytorch_type[type]))
+        delta_default = g.op("Constant", value_t=torch.tensor(1, dtype=sym_help.scalar_type_to_pytorch_type[type]))
+        arange_tensor = g.op("Range", start_default, end, delta_default)
+    elif len(args) == 4:
```

Comment:
What if len(args) == 3?
Does it happen? Like 
aten::arange(Scalar start, Scalar end, Tensor out)

Reply:
Arange signature for 3 args doesn't exist in https://github.com/pytorch/pytorch/blob/348e78b08600e4652e57ed892ba0cf9c7a1661a0/aten/src/ATen/native/native_functions.yaml